docker.sh

```diff
@@ -58,19 +56,21 @@ else
 fi
 echo ""
 
-# build images in this specific order. Images are tagged with latest so other
-# docker images build in this script will use that specifc build.
-for i in base models executor web data thredds docs; do
-    ${DEBUG} docker build \
+# require all of PEcAn to build
+for x in base web docs; do
+    ${DEBUG} docker build --tag pecan/$x:latest --file docker/$x/Dockerfile \
```

Comment:
Similar to above, can we make `latest` here be user-configurable? It might make sense to tie this to `IMAGE_VERSION`, so that the containers have mutually-consistent tags.

Reply:
fixed